homeassistant/components/dlna_dmr/diagnostics.py

```diff
@@ -0,0 +1,15 @@
+"""Diagnostics support for DLNA."""
```

Comment:
Implementation of diagnostics platforms requires 100% test coverage.

Reply:
Yep. I am learning slowly... and now have a proper development environment so should get to it shortly. Thank you for the feedback.